components/brave_wallet/browser/json_rpc_service.cc

```diff
@@ -1696,6 +1696,15 @@ void JsonRpcService::OnGetSPLTokenAccountBalance(
     mojom::SolanaProviderError error;
     std::string error_message;
     ParseErrorResult<mojom::SolanaProviderError>(body, &error, &error_message);
+
+    // Treat balance as 0 if the associated token account is not created yet.
+    if (error == mojom::SolanaProviderError::kInvalidParams &&
+        error_message.find("could not find account") != std::string::npos) {
```

Comment:
string came from https://github.com/solana-labs/solana/blob/master/rpc/src/rpc.rs#L1717

Reply:
can we make it a constant under `solana` namespace in anonymous namespace of this file? Also put comment saying where it came from with permanent link
```c++
namespace {
...
namespace ethereum {
bool IsChainExist(PrefService* prefs, const std::string& chain_id) ...
void ChainIdValidationResponse(...
}
namespace solana {
constexpr char kAccountNotCreatedError[] ="could not find account";
}
}